libbeat/outputs/elasticsearch/docs/elasticsearch.asciidoc

```diff
@@ -124,10 +124,18 @@ The default value is `1`.
 
 ===== `loadbalance`
 
-If set to true and multiple {es} hosts are configured, the output plugin
-load balances published events onto all {es} hosts. If set to false,
-the output plugin sends all events to only one host (determined at random) and
-will switch to another host if the selected one becomes unresponsive.
+When `loadbalance: true` is set, Filebeat sends data if-and-only-if it can send
+to all hosts in the list. Filebeat connects to all hosts in the list and evenly
+distributes the events to them. This has the side effect of blocking publishing
+to all hosts if any one of them is down or exerting back-pressure.
+
+When `loadbalance: false` is set, Filebeat picks a host at random from the list
+and sends data, switching over to other list entries in the event of a failure.
+Filebeat also accepts a time to live (TTL), which causes it to voluntarily
+reestablish a connection to a new host after a time even if it is healthy.
+Otherwise, if a TTL is not set, as long as the first chosen endpoint is available, 
+Filebeat continues to communicate with it and only switches to another endpoint
+in case of a failure.
```

Comment:
```suggestion
When `loadbalance: true` is set, {beatname_uc} sends data if-and-only-if it can send
to all hosts in the list. {beatname_uc} connects to all hosts in the list and evenly
distributes the events to them. This has the side effect of blocking publishing
to all hosts if any one of them is down or exerting back-pressure.

When `loadbalance: false` is set, {beatname_uc}picks a host at random from the list
and sends data, switching over to other list entries in the event of a failure.
{beatname_uc} also accepts a time to live (TTL), which causes it to voluntarily
reestablish a connection to a new host after a time even if it is healthy.
Otherwise, if a TTL is not set, as long as the first chosen endpoint is available, 
{beatname_uc} continues to communicate with it and only switches to another endpoint
in case of a failure.
```

Reply:
Thanks! Fixed via [75892b1](https://github.com/elastic/beats/pull/35271/commits/75892b1edb45c73bf174038e52e7ca205574a9ba) (I fixed it locally since I also needed to adjust the line breaks).